src/unix/notbsd/mod.rs

```diff
@@ -53,7 +53,9 @@ s! {
         pub ai_protocol: ::c_int,
         pub ai_addrlen: socklen_t,
 
-        #[cfg(any(target_os = "linux", target_os = "emscripten"))]
+        #[cfg(any(target_os = "linux",
+                  target_os = "emscripten",
```

Comment:
Late nitpick - couldn't this be:

```
        #[cfg(any(target_os = "linux",
                        target_os = "android",
￼                        target_os = "emscripten",
                        target_os = "fuchsia"))]
        pub ai_addr: *mut ::sockaddr,
```

rather than having the android entry be separate at line 63?